test/components/addressbook/addressbook.controller.js

```diff
@@ -69,13 +85,42 @@ describe('AddressbookController', function () {
       beforeEach(function () {
         storageServiceMock.get = sinon.stub().returns(null)
       })
-
       it('sets controller contacts to empty array on invalid return value', () => {
         ctrl.getContacts()
 
         expect(storageServiceMock.get.calledOnce).to.be.true
         expect(ctrl.contacts).to.deep.equal([])
       })
     })
+
+    context("set up addressbook contact modal", () => {
```

Comment:
This could be wrapped in a sibling block to the `'getContacts'` describe block since its testing a new method. Each method can be described using the method name: `describe('addAddressbookContact', () =>)`